packages/ses/test/console-error-trap.test.js

```diff
@@ -44,47 +48,51 @@ const exitAssertions = (
 };
 
 test('errors reveal their stacks', async t => {
-  t.plan(5);
+  t.plan(6);
```

Comment:
What other change does this number change reflect? I didn't spot it.

Reply:
The change is by adding a single new test to the `exitAssertions` utility function that all of these top-level `test('...')` blocks run, thus incrementing the number of planned assertions in every one of the top-level tests.

Reply:
Resolving.